relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -322,6 +322,13 @@ impl TransactionExtractor<'_> {
                     } else {
                         tags.clone()
                     },
+                    score_profile_version: if is_performance_score {
+                        event
+                            .tag_value("sentry.score_profile_version")
+                            .map(|version| version.to_string())
+                    } else {
+                        None
+                    },
```

Comment:
Not sure if we can simplify this, but maybe something like this works:
```suggestion
                    score_profile_version: is_performance_score.then(|| event.tag_value("sentry.score_profile_version")).map(|version| version.to_string())
```

Reply:
https://github.com/getsentry/relay/pull/3249/commits/451678d1412c22187bdd978450035a0226db4e46
I had to do something slightly different but your suggestion helps make this cleaner, thanks 👍